src/partials/footer-scripts.hbs

```diff
@@ -36,8 +36,15 @@
                 // Create the child element for the title
                 const titleEl = document.createElement("p");
                 titleEl.classList.add("pagefind-modular-list-title");
-                titleEl.innerText = result.meta.title;
                 innerEl.appendChild(titleEl);
+
+                // Create the anchor element
+                const linkEl = document.createElement("a");
```

Comment:
In Chromium, the mouse pointer changes to ☜ when the search dialog is open. It feels like the entire result boxes are clickable, but only the title is. It is better in Firefox, because the mouse pointer only changes on the clickable title.

Reply:
it is also doing that with the current implementation, so I would consider this a separate issue and not deal with it here.